File: module_5/python/validate_input.py
# Assumes that strings with all alphabetical
# characters and no spaces is a "word".
import re

user_input = input('Please type a word: ')

while not bool(re.search('^[A-Za-z]+$', user_input)):
    user_input = input('Your input was not a word... please try again: ')
    
print('Your input was a word!')

File: module_5/r/validate_input.R
# Assumes that strings with all alphabetical
# characters and no spaces is a "word".
validate_input <- function(){
  
  input <- readline("Please type a word: ")

while(!grepl("^[A-Za-z]+$", input)){
  input <- readline("Your input was not a word... please try again: ")
}
  print("Your input was a word!")
}
  
validate_input()
